rotkehlchen/tests/unit/decoders/accounting_settings/test_default_settings.py

```diff
@@ -0,0 +1,271 @@
+
+    gas_event = HistoryBaseEntry(
+        event_identifier=evmhash,
+        sequence_index=0,
+        timestamp=TimestampMS(1628994441000),
+        location=Location.BLOCKCHAIN,
+        location_label=string_to_evm_address('0x4bBa290826C253BD854121346c370a9886d1bC26'),
+        asset=A_ETH,
+        balance=Balance(amount=ONE),
+        notes='Burned 1 ETH for gas',
+        event_type=HistoryEventType.SPEND,
+        event_subtype=HistoryEventSubType.FEE,
+        counterparty=CPT_GAS,
+    )
+    consumed_num = pot.transactions.process(event=gas_event, events_iterator=None)
+    assert consumed_num == 1
+
+    acquisition_event = AssetAcquisitionEvent(
+        amount=ONE,
+        timestamp=Timestamp(1624395186),
+        rate=FVal(1862.06),
+        index=0,
+    )
+    acquisition_event.remaining_amount = ZERO
+    expected_event = ProcessedAccountingEvent(
+        type=AccountingEventType.TRANSACTION_EVENT,
+        notes='Burned 1 ETH for gas',
+        location=Location.BLOCKCHAIN,
+        timestamp=1628994441,
+        asset=A_ETH,
+        free_amount=ZERO,
+        taxable_amount=ONE,
+        price=FVal(3258.55),
+        pnl=PNL(free=ZERO, taxable=FVal(-1862.06)),
+        cost_basis=CostBasisInfo(
+            taxable_amount=ONE,
+            taxable_bought_cost=FVal(1862.06),
+            taxfree_bought_cost=ZERO,
+            matched_acquisitions=[
+                MatchedAcquisition(
+                    amount=ONE,
+                    event=acquisition_event,
+                    taxable=True,
+                ),
+            ],
+            is_complete=True,
+        ),
+        index=1,
+        extra_data={'tx_hash': tx_hex},
+    )
+    expected_event.calculate_pnl(count_entire_amount_spend=True, count_cost_basis_pnl=True)
+    assert pot.processed_events[-1] == expected_event
+
+
+@pytest.mark.parametrize('mocked_price_queries', [prices])
+def test_accounting_spend_settings(pot):
+    """Test that the default accounting settings for spends are correct"""
+    tx_hex = '0x282f632c7a2385d026068122dc415562a1c4e827fd54673671af1d40445d6b58'
+    evmhash = deserialize_evm_tx_hash(tx_hex)
+    # Gain some ETH so that the spending event has something to spend
+    gain_one_ether(transactions=pot.transactions, evmhash=evmhash)
+
+    gas_event = HistoryBaseEntry(
+        event_identifier=evmhash,
+        sequence_index=0,
+        timestamp=TimestampMS(1628994441000),
+        location=Location.BLOCKCHAIN,
+        location_label=string_to_evm_address('0x4bBa290826C253BD854121346c370a9886d1bC26'),
+        asset=A_ETH,
+        balance=Balance(amount=ONE),
+        notes='Send 1 ETH to 0xABC',
+        event_type=HistoryEventType.SPEND,
+        event_subtype=HistoryEventSubType.NONE,
+    )
+    consumed_num = pot.transactions.process(event=gas_event, events_iterator=None)
+    assert consumed_num == 1
+
+    acquisition_event = AssetAcquisitionEvent(
+        amount=ONE,
+        timestamp=Timestamp(1624395186),
```

Comment:
This timestamp comes from the transaction in the method you have right? Maybe you can return it and use it instead of repeating this constant in all the tests